### PR TITLE
Code coverage

### PR DESCRIPTION
**Pull request by LHMarshall**

### CODE REVIEW

**Comment by LHMarshall** [comment]
@alex-konovalov that should be it but it is not running it at the moment I guess because it is in the branch not in the master? Above where all the R-CMD-check's are there should be a code coverage being run too. Not sure how to test without merging...

File: .github/workflows/test-coverage.yaml
Comment (olexandr-konovalov) [inline .github/workflows/test-coverage.yaml:2]: Is this based on https://github.com/r-lib/actions/blob/master/examples/test-coverage.yaml? If so, please add a comment, like in the other `.yaml` file.

File: .github/workflows/test-coverage.yaml
Comment (olexandr-konovalov) [inline .github/workflows/test-coverage.yaml]: Please add a new line at the end of the file (as I explained in another PR this morning, why this is recommended)

File: DESCRIPTION
Comment (olexandr-konovalov) [inline DESCRIPTION:6]: We can leave this as is, but these are unrelated changes in one commit - ideally should be two different commits.

File: .github/workflows/test-coverage.yaml
Comment (olexandr-konovalov) [inline .github/workflows/test-coverage.yaml:5]: In both these places, `master` should be `main`, since this is now used by default for new repositories. This is why the coverage test was not triggered. If you look at https://github.com/r-lib/actions/blob/master/examples/test-coverage.yaml they use both `master` and `main` just to make it work for both conventions.

**Review by olexandr-konovalov — CHANGES_REQUESTED** [review]
@LHMarshall please replace `master` by `main` in two places.

File: DESCRIPTION
Comment (LHMarshall) [reply DESCRIPTION:6]: good to know... you'll get me doing tidy github repo's yet! :)

Comment (olexandr-konovalov) [reply DESCRIPTION:6]: yes, and even furthermore, changing authors is unrelated to the topic of the PR (code coverage) so I'd be reluctant to do "squash and merge" as well...

**Review by olexandr-konovalov — APPROVED** [review]
Nice @LHMarshall - after fixing the branch name, code coverage shows up at https://app.codecov.io/gh/alex-konovalov/tempConverter/pulls - it should be then visible at the main Codecove page for this package after we merge this PR. I am approving it now, but just let me know if you'd like to practice command line rebasing, or should I merge it as is.

**Comment by LHMarshall** [comment]
@alex-konovalov just merge. Is there a cheat sheet somewhere for the command line commands?

File: .github/workflows/test-coverage.yaml
Comment (olexandr-konovalov) [inline .github/workflows/test-coverage.yaml:53]: A minor nitpick, but your editor autoindents after pressing "enter", so instead of adding a line break, you have added a new line with a few spaces...

**Comment by olexandr-konovalov** [comment]
Thanks, done! https://education.github.com/git-cheat-sheet-education.pdf is one of many cheat sheets lying around.

**Pull request merged**
